custom/icds_reports/tasks.py

```diff
@@ -613,11 +613,12 @@ def update_child_health_monthly_table(day, state_ids):
 
 @task(serializer='pickle', queue='icds_aggregation_queue', default_retry_delay=15 * 60, acks_late=True)
 @track_time
-def _child_health_helper(query, params):
-    celery_task_logger.info("Running child_health_helper with %s", params)
+def _child_health_helper(queries):
     with get_cursor(ChildHealthMonthly) as cursor:
-        cursor.execute(query, params)
-    celery_task_logger.info("Completed child_health_helper with %s", params)
+        for query, params in queries:
+            celery_task_logger.info("Running child_health_helper with %s", params)
+            cursor.execute(query, params)
+        celery_task_logger.info("Completed child_health_helper with %s", params)
```

Comment:
Is this intentionally outside the loop?

Reply:
Yeah, we only need it to log completion once per state. If I could find a good way to access the params outside the loop, I would log the start outside as well. Might just add a flag so it only logs the start once per loop, although that seems kludgy.